feat(genapi): update model names in tutorial (#6580)
scaleway/docs-content: pages/generative-apis/reference-content/integrate-with-n8n.mdx

10. Click **Save**.
    The credentials are saved, and an automatic connection test is carried out, displaying the message **Connected tested successfully**. This confirms that n8n can access the `/v1/models` and `/v1/chat/completions` endpoints. 
11. Close the credentials screen to go back to the model parameters.

> 12. Under **Model**, click your chosen model, e.g. `gemma-4-26b-a4b-it`.

Reviewer: ```suggestion
12. Under **Model**, click your chosen model, e.g., `gemma-4-26b-a4b-it`.
```

12. Click **Save**.
    The credentials are saved, and an automatic connection test is carried out, displaying the message **Connected tested successfully**. This confirms that n8n can access the `/v1/models` and `/v1/chat/completions` endpoints. 
13. Close the credentials screen to go back to the **Parameters** tab.

> 14. Under **Model**, click your chosen model, e.g. `qwen3.6-35b-a3b` (other listed models are not supported by Scaleway Generative APIs at the time of writing).

Author: ```suggestion
14. Under **Model**, click your chosen model, e.g. `qwen3-embedding-8b` (other listed models are not supported by Scaleway Generative APIs at the time of writing).
```

> 14. Under **Model**, click your chosen model, e.g. `qwen3-embedding-8b` (other listed models are not supported by Scaleway Generative APIs at the time of writing).

Reviewer: ```suggestion
14. Under **Model**, click your chosen model, e.g., `qwen3-embedding-8b` (other listed models are not supported by Scaleway Generative APIs at the time of writing).
```
